Source/WebCore/page/Quirks.cpp

```diff
@@ -537,22 +537,6 @@ bool Quirks::shouldDispatchedSimulatedMouseEventsAssumeDefaultPrevented(EventTar
     return false;
 }
 
-// maps.google.com https://bugs.webkit.org/show_bug.cgi?id=199904
-std::optional<Event::IsCancelable> Quirks::simulatedMouseEventTypeForTarget(EventTarget* target) const
-{
-    if (!shouldDispatchSimulatedMouseEvents(target))
-        return { };
-
-    // On Google Maps, we want to limit simulated mouse events to dragging the little man that allows entering into Street View.
-    if (isGoogleMaps()) {
-        if (RefPtr element = dynamicDowncast<Element>(target); element && element->getAttribute(HTMLNames::classAttr) == "widget-expand-button-pegman-icon"_s)
-            return Event::IsCancelable::Yes;
-        return { };
-    }
-
-    return Event::IsCancelable::Yes;
```

Comment:
I think we still need to return `Event::IsCancelable::Yes` when `shouldDispatchSimulatedMouseEvents(target)` is true, so we may just need to revise this to:

```
std::optional<Event::IsCancelable> Quirks::simulatedMouseEventTypeForTarget(EventTarget* target) const)
{
    return shouldDispatchSimulatedMouseEvents(target) ? Event::IsCancelable::Yes : { };
}
```

Alternatively, we could remove this entire method from here, but we would need to make more changes to the Safari code to avoid impacting quirks used for desktop iPad sites.

Reply:
@karlcow and I spoke in person, and decided deleting this unused code is a better approach, and will change the Safari bit to match.